decred_wallet/view_controller/Security/SecurityViewController.swift

```diff
@@ -31,6 +31,9 @@ class SecurityViewController: UIViewController, SeedCheckupProtocol, StartUpPass
         btnPassword.backgroundColor = #colorLiteral(red: 0.9449833035, green: 0.9450840354, blue: 0.9490672946, alpha: 1)
     }
     
+    override func viewDidLoad() {
```

Comment:
This overriden function does nothing other than calling the same function on the parent class; please remove it.